lib/screen/collection_pro/collection_pro_screen.dart

```diff
@@ -184,21 +184,12 @@ class CollectionProState extends State<CollectionPro>
                   isShowSearchBar = true;
                 });
               },
-              constraints: const BoxConstraints(
-                maxWidth: 44,
-                maxHeight: 44,
-                minWidth: 44,
-                minHeight: 44,
-              ),
-              icon: Padding(
-                padding: const EdgeInsets.all(0),
-                child: SvgPicture.asset(
-                  'assets/images/search.svg',
-                  width: 24,
-                  height: 24,
-                  colorFilter:
-                      const ColorFilter.mode(AppColor.white, BlendMode.srcIn),
-                ),
+              icon: SvgPicture.asset(
```

Comment:
why `BoxConstraints ` is removed?

Reply:
it doesn't seem to serve any purpose, ui doesn't not change after remove it